tests/dynamic_data/publications/layer_by_used_servers/layer_patch_without_data_test.py

```diff
@@ -0,0 +1,53 @@
+from copy import deepcopy
+import os
+import pytest
+
+from test_tools import process_client
+from tests import EnumTestTypes
+from tests.asserts.final.publication import util as assert_util
+from tests.dynamic_data import base_test
+
+DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+pytest_generate_tests = base_test.pytest_generate_tests
+
+LAYERS = {
+    'main': {},
+}
+
+
+def generate_test_cases():
+    tc_list = []
+    for name, test_case_params in LAYERS.items():
+        all_params = deepcopy(test_case_params)
+        test_case = base_test.TestCaseType(key=name,
+                                           type=EnumTestTypes.OPTIONAL,
+                                           params=all_params,
+                                           marks=[pytest.mark.xfail(reason="Not yet implemented.")]
+                                           if test_case_params.get('xfail') else []
+                                           )
+        tc_list.append(test_case)
+    return tc_list
+
+
+class TestLayer(base_test.TestSingleRestPublication):
+    workspace = 'dynamic_test_workspace_standard_layer'
+
+    publication_type = process_client.LAYER_TYPE
+
+    rest_parametrization = [
+        base_test.LayerByUsedServers,
+    ]
+
+    test_cases = generate_test_cases()
+
+    def test_layer(self, layer, rest_args, rest_method):
+        """Parametrized using pytest_generate_tests"""
+        rest_method(layer, args=rest_args)
+
+        assert_util.is_publication_valid_and_complete(layer)
+
+        self.patch_publication(publication=layer,
+                               args=rest_args)
```

Comment:
In ticket there are mentioned two test cases: access rights and title. I don't see them in this PR, not sure why.